### PR TITLE
Dashboard: Updated Wellbeing template, improved blob imports

### PR DESCRIPTION
**Pull request by mariano-formidable**
## Summary

This PR fixes the Wellbeing masking blobs.  Some were offset from center and the blob itself did not touch all corners of its bounding box.  This is because... I did a bad job importing them the first time 😱 

However, thanks to a nice Illustrator run down from Max, I was able to re-import the Wellbeing blobs perfectly centered and scaled to touch the bounding box 🎉 and this resulted in me having to update our Wellbeing template to adjust for the slight blob position changes that occurred.

## User-facing changes

- The `Wellbeing` template should appear as it did before.
- And 4 new blobs should appear in the Shapes panel of the Editor:
image

## Testing Instructions

Wellbeing Template:
1.) Go to "Explore Templates".
2.) Click on "View" for the `Wellbeing` template.
3.) Make sure each page loads and looks goodly.

New Editor Blobs:
1.) Create a new story (or edit an existing one).
2.) Toggle to Shapes panel.
3.) Confirm that the 4 new blobs appear, and confirm that when added to the scene they're perfectly contained (no extra space) within their bounding box (like the images below).

### CODE REVIEW

**Review by maxyinger — APPROVED** [review]
lookin good!

File: assets/src/dashboard/templates/raw/wellbeing.json [hunk 1]
```diff
@@ -1467,15 +1435,19 @@
           "type": "shape",
           "basedOn": "70cd1c38-9462-47f4-9a12-c9585c16acc4",
           "id": "95a2a69d-7946-4fd2-97ad-7cd47fe87e04",
-          "link": null
+          "link": {
+            "url": "http://example.com/",
+            "icon": null,
+            "desc": "Example Domain"
```
Comment (mariano-formidable) [inline assets/src/dashboard/templates/raw/wellbeing.json:1441]: I also hooked up the links on page 4 of the Wellbeing template.

**Review by BrittanyIRL — APPROVED** [review]
LGTM!

File: assets/src/dashboard/templates/raw/wellbeing.json [hunk 2]
```diff
@@ -1349,7 +1309,11 @@
           },
           "type": "shape",
           "id": "3016e5bf-a0d2-4d9d-82f0-cb169b5e9189",
-          "link": null
+          "link": {
+            "url": "http://example.com/",
```
Comment (swissspidy) [inline assets/src/dashboard/templates/raw/wellbeing.json]: Nit: let's use `https://` everywhere

```suggestion
            "url": "https://example.com/",
```

File: assets/src/dashboard/templates/raw/wellbeing.json [hunk 3]
```diff
@@ -1408,7 +1372,11 @@
           "type": "shape",
           "basedOn": "3016e5bf-a0d2-4d9d-82f0-cb169b5e9189",
           "id": "70cd1c38-9462-47f4-9a12-c9585c16acc4",
-          "link": null
+          "link": {
+            "url": "http://example.com/",
```
Comment (swissspidy) [inline assets/src/dashboard/templates/raw/wellbeing.json]:
```suggestion
            "url": "https://example.com/",
```

File: assets/src/dashboard/templates/raw/wellbeing.json [hunk 4]
```diff
@@ -1467,15 +1435,19 @@
           "type": "shape",
           "basedOn": "70cd1c38-9462-47f4-9a12-c9585c16acc4",
           "id": "95a2a69d-7946-4fd2-97ad-7cd47fe87e04",
-          "link": null
+          "link": {
+            "url": "http://example.com/",
```
Comment (swissspidy) [inline assets/src/dashboard/templates/raw/wellbeing.json]:
```suggestion
            "url": "https://example.com/",
```

Comment (mariano-formidable) [reply assets/src/dashboard/templates/raw/wellbeing.json]: Updated 👍

File: assets/src/dashboard/templates/raw/wellbeing.json (hunk 3 above)
Comment (mariano-formidable) [reply assets/src/dashboard/templates/raw/wellbeing.json]: Updated 👍

File: assets/src/dashboard/templates/raw/wellbeing.json (hunk 2 above)
Comment (mariano-formidable) [reply assets/src/dashboard/templates/raw/wellbeing.json]: 👍

**Comment by mariano-formidable** [comment]
@googlebot I consent

**Pull request merged**